src/os_common/os_fs_inmemory.c

```diff
@@ -263,13 +263,15 @@ __im_fs_size(WT_FILE_SYSTEM *file_system,
 	session = (WT_SESSION_IMPL *)wt_session;
 
 	__wt_spin_lock(session, &im_fs->lock);
+	im_fh = __im_handle_search(file_system, name);
+	__wt_spin_unlock(session, &im_fs->lock);
```

Comment:
Hmm, on reflection, this probably deserves a comment so it doesn't get reverted...